bytereplacement

```diff
@@ -490,3 +490,9 @@ arm9 080550D4 2C
 0012 0223FBD0 81
 0012 0223FBD4 81
 0012 0223FBDC 81
+
+# change ble to b for very fast HP bar
```

Comment:
Add config to toggle fast or slow HP bar

Reply:
alright